lib/homework_1/task_2.js

```diff
@@ -0,0 +1,19 @@
+import {pipeline} from "stream";
+import path from "path";
+import fs from "fs";
+import csv from "csvtojson";
+
+const csvFilePath = path.join(process.cwd(), 'csv', 'nodejs-hw1-ex1.csv');
+
+pipeline(
+    fs.createReadStream(csvFilePath),
+    csv(),
+    fs.createWriteStream('nodejs-hw1-ex2.txt'),
+    (err) => {
+        if (err) {
```

Comment:
1. Close the process once file processing is done. It doesn't make sense to keep it live.
2. Per output format example (https://epam.sharepoint.com/sites/EPAMNode.jsGlobalMentoringProgram/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FEPAMNode%2EjsGlobalMentoringProgram%2FShared%20Documents%2FGeneral%2FAssets%2Fnodejs%2Dhw1%2Dex2%2Etxt&parent=%2Fsites%2FEPAMNode%2EjsGlobalMentoringProgram%2FShared%20Documents%2FGeneral%2FAssets&p=true&originalPath=aHR0cHM6Ly9lcGFtLnNoYXJlcG9pbnQuY29tLzp0Oi9zL0VQQU1Ob2RlLmpzR2xvYmFsTWVudG9yaW5nUHJvZ3JhbS9FVnJ1M2hxQ1JYSk9oOTNGXy10NmV3QUJZSV9Cd3V4ZkpfZXBqcWxBYVdySk9BP3J0aW1lPWo4YTlZcl9mMkVn) column "Amount" should not be present. Please try to filter input data.

Reply:
Done. Thank you.